lib/srv/desktop/rdp/rdpclient/src/rdpdr.rs

```diff
@@ -143,6 +146,28 @@ impl TeleportRdpdrBackend {
         Ok(())
     }
 
+    fn handle_list_readers(
+        &mut self,
+        req: DeviceControlRequest<ScardIoCtlCode>,
+        _call: ListReadersCall,
+    ) -> PduResult<()> {
+        self.write_rdpdr_dev_ctl_resp(
+            req,
+            Box::new(ListReadersReturn::new(
+                ReturnCode::Success,
+                vec!["Teleport".to_string()],
```

Comment:
Should we stick the reader name in a const?

Reply:
Yes, it gets stuck in a const in the next PR in the stack